css/templates/lune/page-vars.css

```diff
@@ -0,0 +1,51 @@
+:root {
+  --content-max-width: 100%;
+
+  --z1: 1; /* body */
+  --z2: 2; /* overlay */
+  --z3: 3; /* sidebar mask */
+  --z4: 4; /* sidebar */
+  --z5: 5; /* header */
+
+  --layout-header-height-m: 4.375rem; /* 70px */
+  --layout-header-height-d: 5.75rem; /* 92px */
+
+
+
+  /* Type */
+  --font-family: "Lato", -apple-system, BlinkMacSystemFont, "Helvetica Neue", "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", Arial, sans-serif;
+  --font-weight-black: 900;
+
+  /* Responsive headings type scale */
+  --h1: 5.625rem; /* 90px */
+  --h2: 5rem; /* 80px */
+  --h3: 3.75rem; /* 60px */
+  --h4: 3.125rem; /* 50px */
+  --h5: 2.1875rem; /* 35px */
+  --h6: 1.375rem; /* 22px */
+
+
+  /* Colours */
+  --color-dark-grey: #202125;
+  --color-dark-grey-2: #33353c;
+
+  --color-light-1: #fff;
+  --color-light-2: #f9f9f9;
+
+  /* Backgrounds */
+  --color-bg-dark-m: var(--color-dark-grey);
+  --color-bg-dark-d: var(--color-dark-grey-2);
+  --color-bg-light: var(--color-light-1);
+  --color-overlay-1: var(--color-light-1);
+  --color-overlay-2: var(--color-light-2);
+
+  /* Foregrounds */
+  --color-font-primary: var(--color-dark-grey);
+  --color-font-secondary: var(--color-font-primary);
+
+  /* Extra breakpoints */
+  /* Breakpoint for very large desktops */
+  @custom-media --breakpoint-vld (min-width: 77.5rem); /* 1240px */
```

Comment:
The design features very large typography for large desktops and so we'll need to make a few changes at this resolution